src/testRunner/unittests/tscWatch/helpers.ts

```diff
@@ -468,6 +468,7 @@ namespace ts.tscWatch {
         if (!builderProgram) return;
         if (builderProgram !== oldProgram?.[1]) {
             const state = builderProgram.getState();
+            const internalState = state as any;

```

Comment:
Why as any ?

Reply:
aah because reusable builder state doesnt have it

Reply:
I think it would read better as:
```ts
const state = builderProgram.getState() as BuilderState;
```

Reply:
Yep, I'll update that. Wasn't aware of the correct type.

Reply:
I had to use `as unknown as BuilderState`